--- passt-2026_05_07.1afd4ed/util.h.orig	2026-05-07 08:06:30.000000000 +0200
+++ passt-2026_05_07.1afd4ed/util.h	2026-05-14 19:40:59.651537981 +0200
@@ -59,8 +59,7 @@
 	assert_with_msg((expr), "ASSERTION FAILED in %s (%s:%d): %s",	\
 			__func__, __FILE__, __LINE__, STRINGIFY(expr))
 #else
-#define assert_with_msg(expr, ...)					\
-	((void)(expr), 0 ? (void)0 : abort_with_msg(__VA_ARGS__))
+#define assert_with_msg(expr, ...) 	((void) 0)
 #endif
 
 #ifdef P_tmpdir
